src/main/java/com/example/school/model/Student.java

```diff
@@ -5,12 +5,15 @@
 import lombok.Setter;
 
 import java.time.OffsetDateTime;
+import java.util.UUID;
 
 @Getter
 @Setter
 @AllArgsConstructor
 public class Student {
+    private UUID id;
     private String name;
     private String surname;
-    private OffsetDateTime startStudying;
+    private OffsetDateTime startDate;
+    private Group grade;
```

Comment:
Change the name of the field from `grade` to `group`.

Reply:
Done